visualization/dashboard/src/Localization/en.json

```diff
@@ -213,7 +217,11 @@
         "numericalComparison": "Operation",
         "_numericalComparison.comment": "label for dropdown containing [greater than, less than, equal to]",
         "minimum":"Minimum",
-        "maximum":"Maximum"
+        "maximum":"Maximum",
+        "min":"Min: ",
+        "max":"Max: ",
+        "uniqueValues":"# of unique values: ",
+        "space":" "
```

Comment:
you can also have loc strings with placeholders, {0}, {1} etc. Constructing multiple words with spaces doesn't always translate well to other languages